modules/next/tests/src/Kernel/Entity/NextEntityTypeConfigTest.php

```diff
@@ -140,4 +140,27 @@ public function testRevalidator() {
     $this->assertSame('path', $revalidator->getId());
   }
 
+  /**
+   * Tests config dependency calculation.
+   */
+  public function testConfigDependencies(): void {
+    $blog_site = NextSite::create([
+      'id' => 'blog',
+    ]);
+    $blog_site->save();
+
+    // Create entity type config.
+    /** @var \Drupal\next\Entity\NextEntityTypeConfigInterface $entity_type_config */
+    $entity_type_config = NextEntityTypeConfig::create([
+      'id' => 'node.page',
+      'site_resolver' => 'site_selector',
+      'configuration' => [
+        'sites' => [
+          'blog' => 'blog',
+        ],
+      ],
+    ]);
+    self::assertEquals([], $entity_type_config->getDependencies());
```

Comment:
Am I misreading this assertion?

If the purpose of `calculateDependencies()` is to add the proper dependencies to the config, why is the test asserting that `getDependencies()` is an empty array? How does this test that dependencies are added properly?

Reply:
Good question. I can't remember if I left this here to show the test was still failing or what... Or if I wrote the test first and then the fix?

I'll follow back up on this today